inspire/modules/formatter/format_templates/Default_HTML_detailed.bft

```diff
@@ -3,8 +3,8 @@
 <div class="detailed_record_info">
 
 <div style="text-align:center; margin-bottom: 10px">
-<BFE_INSPIRE_TITLE_BRIEF prefix="<big><big><strong>"
-suffix="</strong></big></big>" force_title_case="yes"
+<BFE_INSPIRE_TITLE_BRIEF prefix="<big><big><strong><div class='editable' field-value='title.title'>"
```

Comment:
Can you change `field-value -> data-field` to conform with HTML standard?

Also remember to change any affected selectors in the Javascript.